### ISSUE TITLE
Gesture parts do not function as intended if set to a non-vanilla activity

### ISSUE THREAD

**Issue by Logzombie** [CapsAdmin/pac3#1121]
This is an issue that cropped up a while ago while I was updating my Left 4 Dead 2 PACs. I've had to recompile the models again just to set their activities from their defaults to stuff like ACT_GESTURE_RANGE_ATTACK1 from base HL2. Other activities from Left 4 Dead itself caused the animation on the gesture part to not play at all.

Side note: This seemed to completely function as intended on singleplayer only, or hosting clients. Which raised an eyebrow whenever I tried to figure out what on earth was happening, and why Caps had trouble attempting to reproduce it when I first let him know. Possible this was changed by the refactor a while ago, but it seems strange that it would require a vanilla activity to function compared to prior when it didn't seem to matter.

**Comment by thegrb93** [comment]
Yeah I remember the model loader's activity mutating code was changed a bit. I don't know anything about source engine's woes with this issue though

**Comment by thegrb93** [comment]
https://github.com/CapsAdmin/pac3/blob/de882f792cf9741900e0b5ed3cc62cb04c801c57/lua/pac3/core/shared/util.lua#L710-L713
